### PR TITLE
[libmediainfo] Update to 20.08

### PR DESCRIPTION
**Pull request by g-maxime**

### CODE REVIEW

File: ports/libmediainfo/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO MediaArea/MediaInfoLib
-    REF v20.03
-    SHA512 c5d3444c8694ca68ee08f97f387cef3aefd9fbb23623b643a6daf9ed7d247521f1291a8a13c9088b31be9a9d594ca772d3125d6eb3d3770bee1f7c50b3b23c07
+    REF v20.08
+    SHA512 271742af2b2407c04a0555d851fc9ef5590326f4101413ef2092d0a0b99e8367d01bb6442464d171b582b00bb2f45edb9bc9867e74a8d14daba99e2416bc08f3
     HEAD_REF master
     PATCHES vcpkg_support_in_cmakelists.patch
 )
```
Comment (NancyLi1013) [inline ports/libmediainfo/portfile.cmake:8]: Could you please update

`OPTIONS -DBUILD_ZENLIB=0 -DBUILD_ZLIB=0`

as

```
OPTIONS 
    -DBUILD_ZENLIB=0 
    -DBUILD_ZLIB=0
```
?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — COMMENTED** [review]
Could you please also bump the `version `in CONTROL file?

File: ports/libmediainfo/portfile.cmake (hunk 1 above)
Comment (g-maxime) [reply ports/libmediainfo/portfile.cmake:8]: Done. Thanks.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**